FogCreek/Glitch-Community
# Persistent papaya: Fix ssr errors on the vscode auth page

## Links
https://persistent-papaya.glitch.me/
https://app.clubhouse.io/glitch/story/4201/the-vscode-auth-page-fails-to-server-side-render

## Changes:
- Add ssr tests for the vscode auth page and the about pages
- Fix the ssr issue in /vscode-auth
- Switched from mocha-jsom to jsdom-global

## How To Test:
- Run `pnpm run test` and make sure the tests pass
- Try going to /vscode-auth and make sure there are no ssr errors in the log
